### PR TITLE
Py37

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

**Comment by mrocklin** [comment]
I've communicated the secrets to Derek in side channels

**Comment by mrocklin** [comment]
Whoops, wrong thread

**Comment by ocefpaf** [comment]
Needs https://github.com/conda-forge/toolz-feedstock/pull/12

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

File: .appveyor.yml [hunk 1]
```diff
@@ -12,9 +12,6 @@ environment:
     - CONFIG: win_c_compilervs2008python2.7
       CONDA_INSTALL_LOCN: C:\Miniconda36-x64
 
-    - CONFIG: win_c_compilervs2015python3.5
```
Comment (ocefpaf) [inline .appveyor.yml:15]: I was hoping @mariusvniekerk would let this one pass ;-p

**Pull request merged**